### DISCUSSION TITLE
希望动态页长图片缩小后可以回到原动态

### DISCUSSION THREAD

**Discussion by yongrenx** [the1812/Bilibili-Evolved#4029] [功能建议 / Ideas]
动态页的图片点开前不知道有多长，有时候就会往下拉很大一截，希望缩小后能跳回到原动态

**Comment by the1812** [comment]
连续多张图, 看了一张很长的, 下一张不长也会有类似的问题
感觉可以把图片限制在一屏内并且可滚动

**Reply by the1812** [reply]
好吧, 做成滚动会有很多奇奇怪怪的问题, 还得是跳回
